app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+  <div class="container">
+    <img class="user-img" src=<%= user.photo %> alt="photo of user">
+    <div class="user-info">
+    <h2><%= user.name %></h2>
```

Comment:
- [ ] Clicking on each user should take you to their show page as per requirements. Currently, I have to manually type in the URL i.e `http://127.0.0.1:3000/users/1` in order to access the user's show page. Please fix this 👍

Reply:
@codepantha Do you have any suggestions?